### PR TITLE
Updated OKE cluster creation scripts to use k8s 1.24.X version

### PR DESCRIPTION
**Pull request by marinakog**
Updated OKE cluster creation scripts to use k8s 1.24.X version
https://build.weblogick8s.org:8443/job/wko-oke-nightly-parallel/411/

### CODE REVIEW

File: Jenkinsfile.oke [hunk 1]
```diff
@@ -62,15 +62,17 @@ pipeline {
...
         string(name: 'KUBE_VERSION',
                description: 'kube version for oke cluster',
-               defaultValue: '1.23.4'
+               defaultValue: '1.24.1'
         )
         string(name: 'IMAGE_ID',
                        description: 'oci image id for node pool, find image OCID for your region from https://docs.oracle.com/iaas/images/',
-                       defaultValue: 'ocid1.image.oc1.phx.aaaaaaaaiwr25nhuo5vfiobwlsmj6qrab6xhzzlqyl7cwmmzeq6mck3w57va'
+                       defaultValue: 'ocid1.image.oc1.phx.aaaaaaaa6med26u7lkdeyz6mjwaepakor5bp2y24lwr4x6adn6izidipbbsa'
+                       //8.0defaultValue: 'ocid1.image.oc1.phx.aaaaaaaan7lpv4nxnx4r772csco2nzre3owzqdebf75ty6mlbp562vn6c45q'
```
Comment (xiancao) [inline Jenkinsfile.oke:70]: Do you want to leave this line here?

**Review by xiancao — APPROVED** [review]
minor comments

File: Jenkinsfile.oke [hunk 2]
```diff
@@ -62,15 +62,17 @@ pipeline {
...
+            
                 )
         string(name: 'KUBECTL_VERSION',
                        description: 'kubectl version',
-                       defaultValue: '1.23.4'
+                       defaultValue: '1.24.8'
```
Comment (ankedia) [inline Jenkinsfile.oke:75]: Should we use same version for `kubectl` and `kube`?

**Review by ankedia — APPROVED** [review]
LGTM.

File: Jenkinsfile.oke (hunk 1 above)
Comment (marinakog) [reply Jenkinsfile.oke:70]: yes , this is for Oracle image with Linux8.0

File: Jenkinsfile.oke (hunk 2 above)
Comment (marinakog) [reply Jenkinsfile.oke:75]: kubectl version is based on downloaded executables in OCI, for now we have 1.24.0 and 1.24.8 available

**Pull request merged**